README.md

```diff
@@ -1,12 +1,91 @@
+We utilized a subset of a dataset originally compiled from 680,333 participants, ranging in age from 7 to 89, for our study. These individuals completed an English grammar quiz and provided demographic information, as well as details about their language backgrounds and the countries they have lived in. This dataset was central to the research paper "A critical period for second language acquisition: Evidence from 2/3 million English speakers" by Joshua K. Hartshorne, Joshua B. Tenenbaum, and Steven Pinker. The study focused on understanding the peak age for grammar-learning ability, observing its maintenance during childhood, and its notable decline in late adolescence. The authors performed various analyses and compared multiple models to identify age-related patterns in language acquisition. The complete dataset is accessible at [https://osf.io/pyb8s/](https://osf.io/pyb8s/). For the initial phase of our project (Milestone 1), we only used 30% of the total dataset. The script for sampling this subset is available in 'src/Sampling from dataset.ipynb'. In future project stages, we plan to consider incorporating the entire dataset. This rich dataset allows for a multi-faceted analysis of language learning patterns.
+
+---
+
+## 👥 Team Members
+
+- Salva
+- Atabak
+- Nando
+- Rachel
+
+---
+
+## 💻 Getting Started
 
 ### Dependencies
 
-- Need to have `conda` installed, this is because we are using a conda environment to manage our dependencies.
+- Ensure you have `conda` installed as we use a conda environment to manage our project dependencies.
+- Python and packages listed in [environment.yml]
 
 ### Installing
 
-- Clone the repo
-- Run `conda env create -f environment.yml` to create the conda environment
+- Clone the repository:
+
+```bash
+git clone [https://github.com/UBC-MDS/522-workflows-group-18]
+cd 522-workflows-group
+```
+
+- Create and activate the conda environment:
+
+```bash
+conda env create -f environment.yml
+conda activate 522
+```
+
+---
+
+## 🚀 Usage
+
+To dive into our analysis:
+
+1. Launch Jupyter Notebook:
+
+```bash
+jupyter notebook English Language Learning Ability Prediction.ipynb
+```
+
+2. Navigate through the notebook to view our data analysis process, model training, and predictions.
+
+---
+
+## 🔍 Methodology
+
+We employ a comprehensive approach:
+
+1. **Data Preprocessing:** Cleaning, normalization, and transformation of data to ensure quality and consistency.
+2. **Exploratory Data Analysis:** Utilizing statistical techniques and visualization tools to uncover trends and patterns.
+3. **Model Development:** TO BE ADDED
```

Comment:
This still says TO BE ADDED

Reply:
Thank you for mentioning that, I updated that one